### PR TITLE
ADDED: expose method 'logConfig'

### PR DESCRIPTION
**Pull request by cdancy**
Addresses issue #156

### CODE REVIEW

**Comment by cdancy** [comment]
@bmuschko second look? I was getting a bit tired of specifying the fully qualified name for LogConfig. Not exactly sure why I did not do this in the first place. Perhaps I was still a bit wet behind the ears.

File: src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy [hunk 1]
```diff
@@ -159,6 +159,10 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
         imageId
     }
 
+    void logConfig(String type, Map<String, String> config) {
+        logConfig = LogConfig.newInstance(type: type, config: config)
```
Comment (bmuschko) [inline src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy]: No need to use the `newInstance` method as there aren't any constructors defined. You can just use `new LogConfig(type: type, config: config)`.

File: src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy [hunk 2]
```diff
@@ -16,5 +16,6 @@ class InspectContainerResponseHandler implements ResponseHandler<String, Object>
         logger.quiet "VolumesFrom : $container.hostConfig.volumesFrom"
         String exposedPorts = container.config?.@exposedPorts ? container.config.exposedPorts : '[]'
         logger.quiet "ExposedPorts : $exposedPorts"
+        logger.quiet "LogConfig : $container.hostConfig.logConfig.type.type"
```
Comment (bmuschko) [inline src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy]: Is `type.type` correct? Shouldn't it just be `$container.hostConfig.logConfig.type`?

**Comment by bmuschko** [comment]
@cdancy See my comments.

File: src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy (hunk 2 above)
Comment (cdancy) [inline src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy]: The first 'type' returns an Enum while the second 'type' returns the string representation of that given enum.

https://github.com/docker-java/docker-java/blob/master/src/main/java/com/github/dockerjava/api/model/LogConfig.java

File: src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy (hunk 1 above)
Comment (cdancy) [inline src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy]: Fixing now.

File: src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy (hunk 2 above)
Comment (bmuschko) [inline src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy]: Ah, never mind. I thought it was the `LogConfig` below. Thanks for clarifying.

**Comment by bmuschko** [comment]
@cdancy LGTM

**Comment by cdancy** [comment]
@bmuschko thanks. Merging now.

**Pull request merged**